torch/csrc/api/include/torch/ordered_dict.h

```diff
@@ -193,6 +193,16 @@ class OrderedDict<Key, Value>::Item {
   /// Constructs a new item.
   Item(Key key, Value value) : pair_(std::move(key), std::move(value)) {}
 
+#if defined(__CUDACC__) && (__CUDACC_VER_MAJOR__ < 11) && defined(_MSC_VER)
```

Comment:
Excuse my innocence but I don’t quite get it why CUDA version matters here?

Reply:
You may know more context here. https://github.com/pytorch/pytorch/pull/47524#issuecomment-736018820